engine/admin/logging/awslogs.md

```diff
@@ -77,6 +77,115 @@ $ docker run --log-driver=awslogs \
 > **Note**:
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.
 
+### awslogs-multiline-pattern
+
```

Comment:
Does this need `{% include edge_only.md section"option" %}` or did the feature make it into the 17.06.0 RC?